requirements.txt

```diff
@@ -4,5 +4,6 @@ certifi==2021.5.30
 dbus-python==1.2.16
 gunicorn==20.1.0
 hm-hardware-defs==0.1.5
-requests==2.25.1
+hm-pyhelper==0.2
+requests==2.26.0
```

Comment:
Out of curiosity, this upgrade wasn't required was it?

Reply:
It was because i set this as the minimum version in the helper.